ui/job-view/App.jsx

```diff
@@ -65,15 +64,11 @@ const getWindowHeight = function getWindowHeight() {
   return windowHeight - navBarHeight;
 };
 
-const store = configureStore();
```

Comment:
When the provider was moved to the top level App, this caused problems since we've essentially added another store. Removing this and adding an action to clear transient notifications fixed that bug.